index.html

```diff
@@ -18,12 +18,17 @@
         <header class="head">Uthman</header>
         <i class="fa-solid fa-bars menu-bar"></i>
     </div>
+    <nav class="nav">
+        <a href="#" class="nav-link">Portfolio</a>
+        <a href="#" class="nav-link">About</a>
+        <a href="#" class="nav-link">Contact</a>
```

Comment:
- [ ] Kindly put these links in a `ul` where each link is wrapped in `li` for semantic reasons 👍